balancer_pool_states.py

```diff
@@ -163,34 +183,34 @@ def compute_balancer_pool_states():
     balancer_pool_states.append(
         {
             "block": START_BLOCK,
-            "COW": current_COW,
-            "WETH": current_WETH,
-            "time": current_time,
+            TOKEN1: CURRENT_TOKEN1,
+            TOKEN2: CURRENT_TOKEN2,
+            "time": CURRENT_TIME,
         }
     )
     for t in range(n + m):
         if i >= n:
             for t in balancer_pool_liquidity_changes[j:]:
-                current_WETH += t["WETH"]
-                current_COW += t["COW"]
+                CURRENT_TOKEN1 += t[TOKEN1]
```

Comment:
Constants must not be changed.
```suggestion
                current_token1 += t[TOKEN1]
```
and the same for token 2 and all other places where `CURRENT_TOKENx` is updated.